tests/test_system.py

```diff
@@ -0,0 +1,71 @@
+import platform
+import socket
+import sys
+
+import pytest
+
+from iopipe import system
+
+
+def test_read_arch():
+    arch = system.read_arch()
```

Comment:
This can be simplified, no assignment is needed if you use the `system.read_arch` on the left of the test below

Reply:
This comment applies to remaining tests in this file as well.

Reply:
Ok, revised those tests.